### ISSUE TITLE
The input value remains in the TextField even after the dialog is closed.

### ISSUE THREAD

**Issue by stoicjone** [jonatanklosko/material-ui-confirm#89]
First of all, I admire and applaud this very simple and great library. Thanks to you, I am using it very well.

But there is one problem. If you close and reopen the dialog box while entering the `confirmationKeyword`, the `TextField` retains the previously entered value. I don't think this behavior is very helpful to UX.

```typescript
import { useConfirm } from 'material-ui-confirm';

// ...
const confirm = useConfirm();

confirm({
  title: 'Check reservation information',
  content: 'Please check your reservation information again.',
  confirmationKeyword: 'I have confirmed the contents and agree to the reservation',
  confirmationKeywordTextFieldProps: {
    size: 'small',
    placeholder: 'I have confirmed the contents and agree to the reservation',
    helperText: 'After confirming your reservation information, please enter the same information as above.',
  },
})
```
![스크린샷 2024-02-06 092926](https://github.com/jonatanklosko/material-ui-confirm/assets/133740820/ac266959-03d4-4bb6-a549-9adaf1395b5e)

Is there a way to initialize the TextField input each time the Dialog is closed? What am I missing?

**Comment by jonatanklosko** [comment]
Fixed in 0f27e189d70548e5ef0c1230a9fd77bd0b67f9c5, released in v3.0.11.

**Comment by stoicjone** [comment]
Wow, you fixed it really quickly. I confirmed that it works as expected. Thank you. Have a good day.
